libp2p/RLPxHandshake.cpp

```diff
@@ -28,6 +28,8 @@ using namespace dev;
 using namespace dev::p2p;
 using namespace CryptoPP;
 
+boost::random_device s_paddingEngine;
```

Comment:
Hm, I think `boost::random_device` is not thread safe. Furthermore, I'm not sure if it is possible to e.g. deplete the entropy pool (and thus possibly degrade the security of more critical systems) by sending massive connection attempts.